java/code/src/com/redhat/rhn/common/db/datasource/xml/Package_queries.xml

```diff
@@ -1069,9 +1070,14 @@ SELECT  n.id AS id,
   join rhnPackageUpgradeArchCompat puac
     on puac.package_arch_id = sp.package_arch_id
    and puac.package_upgrade_arch_id = latest.package_arch_id
+  JOIN rhnErrataPackage ep
+  ON latest_p.id = ep.package_id
```

Comment:
There are packages which are not part of an errata. A normal JOIN would ignore them.
I think we need a LEFT JOIN